main.py

```diff
@@ -489,6 +489,14 @@ def post(self):
         self.response.write(json.dumps({}))
 
 class ClassHandler(BaseHandler):
+    def gen_opted_in_table(self):
+        table = {}
+        opted_in = db.GqlQuery("SELECT * FROM User WHERE share_photo = TRUE")
+        for student in opted_in:
+            table[student.email] = student.share_photo
```

Comment:
student.share_photo will always be true, so you could just use True here, or use a Python set instead of a dictionary.